rosidl_core_runtime/README.md

```diff
@@ -0,0 +1,13 @@
+# rosidl_core_runtime
```

Comment:
a top level readme and license file would be nice as well

Reply:
There is already a top-level license file (I initialized the repository with it).

I've added/update READMEs now: [1d9ac52](https://github.com/ros2/rosidl_core/pull/1/commits/1d9ac528029211bc9dc8abb954cfbf09519ee67f)

Reply:
> There is already a top-level license file (I initialized the repository with it).

Sorry, didn't see that.

> I've added/update READMEs now: [1d9ac52](https://github.com/ros2/rosidl_core/pull/1/commits/1d9ac528029211bc9dc8abb954cfbf09519ee67f)

Thanks!